Kiosk/Auction Listings/ListingsViewController.swift

```diff
@@ -44,7 +44,7 @@ class ListingsViewController: UIViewController {
 
         // Set up development environment.
 
-        if APIKeys.sharedKeys.stubResponses {
+        if APIKeys.sharedKeys.stubResponses && AppSetup.sharedState.isTesting == false {
```

Comment:
As a future refactor thought, if this logic isn't strictly specific to the view, I like the idea of encapsulating it in a "config model" of some sort. That way, if anything else needs to perform a similar mode check, the actual logic is all in one spot.